lib/tests/test_conversion.py

```diff
@@ -52,6 +60,7 @@ def test_coordinates_to_decimal_edgecases(arcminutes, expected_easting, expected
     assert math.isclose(northing, expected_northing, abs_tol=0.00001)
 
 
+@pytest.mark.filterwarnings("ignore::FutureWarning")
```

Comment:
Still not happy about those ignores. I am sure we will forget about them.

Reply:
OK, I'll keep the pytest log cluttered with them, so that it incentivises us to fix it ;)